src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$parts:
```

Comment:
```suggestion
$colors:
```

Reply:
it is not just colors, but also the part numbers combined